manifests/backup.pp

```diff
@@ -1,7 +1,19 @@
+# Backup PostgreSQL
+#
+# @summary Backup PostgreSQL
+
 class pe_databases::backup (
-  Array[Hash] $databases_and_backup_schedule = [
+  Array[Hash] $databases_and_backup_schedule =
+  [
     {
-      'databases' => ['pe-activity', 'pe-classifier', 'pe-postgres', 'pe-rbac', 'pe-orchestrator'],
+      'databases' =>
+      [
+        'pe-activity',
+        'pe-classifier',
+        'pe-orchestrator',
+        'pe-postgres',
+        'pe-rbac',
+      ],
```

Comment:
should probably add pe-inventory for versions that have it :(

Reply:
I wrote code for this, but adding a database will trigger a new cron job and unmanage the old as per the note in the README since the cron job titles include the array of database names 

:(

```
Notice: /Stage[main]/Pe_databases::Backup/Cron[puppet_enterprise_database_backup_[pe-activity, pe-classifier, pe-inventory, pe-orchestrator, pe-postgres, pe-rbac]]/ensure: current_value 'absent', should be 'present'
```

Reply:
I created a workaround for the default backup change triggering a new cron job and unmanaging the old. This should be good.